libs/ui/src/components/atoms/Switch/Switch.tsx

```diff
@@ -73,7 +73,7 @@ export const Switch = React.forwardRef(
           onCheckedChange={onCheckedChange}
           disabled={disabled}
         >
-          <SwitchSlider className={classes} ref={ref} />
+          <SwitchSlider ref={ref} />
         </SwitchBase>
         <LabelContainer>
           <Label id={id} className={classes}>
```

Comment:
We should make a ticket to refactor Switch to use Label Component. 
I don't want to keep bloating this ticket.

Reply:
Agreed.